lua/lspsaga/diagnostic/show.lua

```diff
@@ -232,6 +232,8 @@ function sd:toggle_or_jump(entrys_list)
     end
     api.nvim_win_close(self.winid, true)
     local ln, col, bn = unpack(vim.split(info, ':'))
+    ln = ln - 1
```

Comment:
there is string type i think

Reply:
Thank you for reminding me. Now I have added some type checks